src/controllers/mainMenuController.js

```diff
@@ -8,30 +8,30 @@
  * @constructor
  */
 /* global Ioc, StringUtils, punycode */
-var DetailedMenuController = function ($, wot, localization, gmApi, settings) { // jshint ignore:line
+var DetailedMenuController = function($, wot, localization, gmApi, settings) { // jshint ignore:line
     var contentDocument = null;
     var iframeCtrl = null;
     var domain = null;
+    var FILTERING_STATE_LS_PROPERTY = 'Adguard_Filtering_State';
```

Comment:
It'd be better to have some generic name like `__agfstate` just to not highlight ourselves in the local storage.